lib/command.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
```

Comment:
really? gonna look it up. it did throw errors at me when I accidentally used a global